### ISSUE TITLE
[DESIGN] Collections

### ISSUE THREAD

**Issue by jesicarson** [participedia/api#771]
rough notes on collections so far...
- specific to projects, ie: Government of Canada, or SFU Strengthening Canadian Democracy, or even specific Participedia collections such as "Top 10 Most Read Cases" or whatever we want
- entries labelled as part of a particular collection via the data form 
- possibly problematic if publicly accessible in the form, as general public may not understand what a collection is
- also problematic if admin-only, because the purpose of a collection, ie for the Privy Council/GOV Canada example, is for various authors from different levels of government to be able to have an account to publish and add to a collection
- collections (sets of results) accessed via search filters 

The main issue here is to think through how / who is able to assign an entry to a collection... public access vs admin-only, vs some other level of permissions?

**Comment by jesicarson** [comment]
Discussed the above with @ascott and @paninee, and we agree that the entire form is already built on trusting the community to enter things as they deem correct, therefore we can leave this field open to the public and trust that they won't misuse it. It's also a fact that our editing team reviews all entries and can help watch for mislabeling of collections. Plus, we can include explanatory text outlining what we mean by collections, ie: special uses of Participedia by collaborators who are doing this type of work, don't use this field if you're not one of them, thanks very much, etc.

**Comment by jesicarson** [comment]
First round of mockups for the collections search filter:

Option 1 - Each collection only has one value
![Collections mockup 1 - GOVCA top level](https://user-images.githubusercontent.com/13560530/67121142-a57c6d80-f19f-11e9-8ff2-81675c5e21ce.png)

Option 2 - each collection has one main value, plus sub-values determined by the collaborator (ie: Privy Council Office suggested the values for the Government of Canada collection shown here)
![Collections mockup 2 - GOVCA drill down](https://user-images.githubusercontent.com/13560530/67121143-a6150400-f19f-11e9-8425-f6de10806747.png)

**Comment by jesicarson** [comment]
Update Oct 31 - PCO is interested in testing OPTION 1 above, with "Government of Canada" as the collection label. @ascott @paninee this is next up after search filters for methods / orgs. 

@scottofletcher we might want to make a couple collections to add to this, ie: participedia team entries, or something else, maybe a theme? think about it. 

Amber also pointed out that a future version of collections may need a verification step, and some sort of badge for accounts/entries of public interest ([a-la-twitter](https://help.twitter.com/en/managing-your-account/about-twitter-verified-accounts))

**Comment by scottofletcher** [comment]
RE collections: yes, PPedia team entries would make sense. others to add would be: 
- Sciencewise
- Coady student entries
- South America or other regions that can't be easily accessed on the map

RE verification: it was my understanding that only admins would be able to create collections, so maybe we could have a contact links on the landing page for collections

**Comment by jesicarson** [comment]
Only admins can create a collection (only Devs at the moment actually), but anyone can add any entry to any collection (trust system). There's no landing page, it's going to be a search filter called "Collections" so it will produce a set of results on the homepage like all the other search filters (and can be used in conjunction with other filters). Good point though, about being able to request a collection - this could be included in the explanatory text for the new field. Which reminds me, that needs to be drafted asap!

**Comment by jesicarson** [comment]
**[form field text, cases, full version, below narrative]**

[label] Collection

[explanatory text] If you are a representative of a project or organization that has been identified as a "collection" on Participedia, please select it here. If your entry is not part of a collection, please leave this field blank. 

[info modal] Collections are suited to projects and organizations that are connected to multiple initiatives of public participation, often with a variety of geographic locations and/or primary organizers. The "Collections" search filter on the homepage allows these connected entries to be viewed as a single set of results, with a custom URL that can serve as a landing page for projects on Participedia. If your project is suited to a collection, or if you require further information, please contact us at info@participedia.net. We kindly ask that only representatives of projects identified as collections use this field.

[placeholder text] Select a collection

[values]
- Participedia Team
- Government of Canada
- Sciencewise
- Coady Students

**Comment by jesicarson** [comment]
Question - do we need collections for methods and orgs? or are we only starting with cases for now? thanks.

**Comment by jesicarson** [comment]
and then i suppose this will need to be verified for French in phraseapp once its live

**Comment by scottofletcher** [comment]
this looks great! I guess we could also add USoton Students (Matt) and U Arkansas Students (Robert/John)

**Comment by scottofletcher** [comment]
> do we need collections for methods and orgs?

I don't think so - the 'General Types of Methods' serves as a sort of 'collection' (since they're broad categories). organization entries are not a priority on PPedia and are almost all stubs or partials (and in need of editing)

**Comment by ascott** [comment]
> Question - do we need collections for methods and orgs? or are we only starting with cases for now? thanks.

@scottofletcher @plscully we should determine this before we start on the dev work. should we implement Collections for cases only? or can a collection include cases, orgs and methods?

**Comment by jesicarson** [comment]
Ok, so cases only.

**Comment by ascott** [comment]
@jesicarson @scottofletcher to confirm a case can be added to multiple collections, is that correct? I think this makes the most sense, as I can see a scenario where we may want to add a case to the Government of Canada collection, but perhaps also to the Top 10 most Read collection at some point down the road, but wanted to confirm with you both.

**Comment by jesicarson** [comment]
I was thinking just one collection per entry, but i see your point...  i think i'm ok with it either way. but maybe lets keep the language as is, which implies you should only select 1 collection.

**Comment by scottofletcher** [comment]
yes, that makes sense to me as long as there's some kind of admin verification process. I could see users abusing the Top 10 Most Read collection (we saw the same thing with the completeness level field on the old site - users would mark theirs complete, ostensibly to get more views)

**Comment by ascott** [comment]
@scottofletcher there will not be an admin verification process for this field, it will be open to all users to edit. Can you think of other instances where a case might need to be included in multiple collections?

**Comment by scottofletcher** [comment]
oh plenty, I mean technically if we have a collection for PPedia Team, all of the Sciencewise entries also go in it because they were entered by Martin (Graham's RA). then if we have collections for different regions or levels of government, there could be lots of cross-over

**Comment by ascott** [comment]
ok, let's make it a multi select field then to allow multiple collections per case.

**Comment by jesicarson** [comment]
A future consideration, brought up by the Privy Council folks, is to have a notification system to let people know when an entry that's part of a collection has been edited, or if that label was removed or added. But they're also fine to give it a try and see how it goes for now.

**Comment by scottofletcher** [comment]
I think that was something we heard from a number of people at the conference too and not just in relation to collections, but to any entry that they submitted or that they sign up to receive alerts from

**Comment by plscully** [comment]
Hi all -- I like where this is going. I agree that for now we should limit collections to cases, and that it should be possible for a case to appear in more than one collection. I created this g-doc to respond to the draft of the form field explanatory text https://docs.google.com/document/d/1kmv3EhA9UT0soeTqdnuyJP04P70AUhLVZcMU4lxartM/edit
